cost-analyzer/values.yaml

```diff
@@ -615,6 +619,7 @@ clusterController:
   enabled: false
   image: gcr.io/kubecost1/cluster-controller:v0.0.2
   imagePullPolicy: Always
+  fqdn: TODO-some-fqdn-example
 
```

Comment:
TODO: I'd like some help in giving some suitable examples here!

Reply:
Sorry, didn't see which block this was under. on this particular line you'll want to add the default servicename for the cluster-controller-- just something like kubecost-cluster-controller.kubecost.svc.cluster.local.

Reply:
thanks! just to check - a port as well?

Reply:
Thanks! I've added them commented-out by default and updated the examples.